portfolio/src/main/webapp/script.js

```diff
@@ -50,4 +49,23 @@ function addCommentToDom(comment) {
   console.log('Adding comment to dom: ' + comment);
   const commentContainer = document.getElementById('comment-container');
   commentContainer.innerText = comment;
-}
\ No newline at end of file
+}
+
+/** Adds all comments to list */
+function getAllComments() {
```

Comment:
I think this name implies that is a function that would return a list of comments instead of populating the commentList element. Maybe call it pouplateCommentThread, or something like that